### PR TITLE
style(api): Revise execute/simulate reference docstrings

### PR DESCRIPTION
**Pull request by SyntaxColoring**
## Overview

Minor adjustments to the reference docstrings in `opentrons.simulate` and `opentrons.execute`.

## Test Plan and Hands on Testing

I've skimmed the generated docs and made sure this doesn't introduce any flagrant formatting problems.

## Changelog

* Note, for a couple of arguments, that they are only intended for internal Opentrons use.
* Adjust indentation so we don't have to describe every parameter with ~50-character-long lines. (No user-visible difference, just changing the formatting in the .py file.)
* A couple other tiny fixups, like `Python` instead of `python`.

## Review requests

Is `chore_release-8.0.0` the correct merge target?

## Risk assessment

No risk.

### CODE REVIEW

**Review by sfoster1 — APPROVED** [review]
Yeah, that looks like a good set of changes

File: api/src/opentrons/execute.py [hunk 1]
```diff
@@ -313,18 +313,18 @@ def execute(
 
     :param protocol_file: The protocol file to execute
     :param protocol_name: The name of the protocol file. This is required
-                          internally, but it may not be a thing we can get
-                          from the protocol_file argument.
+        internally, but it may not be a thing we can get
+        from the protocol_file argument.
```
Comment (ecormany) [inline api/src/opentrons/execute.py]:
```suggestion
        from the ``protocol_file`` argument.
```
Mono for an argument name.

File: api/src/opentrons/execute.py [hunk 2]
```diff
@@ -142,23 +142,23 @@ def get_protocol_api(
...
+        as a ``protocols.types.APIVersion``
+        (``APIVersion(2, 0)``).
     :param bundled_labware: If specified, a mapping from labware names to
-                            labware definitions for labware to consider in the
-                            protocol. Note that if you specify this, _only_
-                            labware in this argument will be allowed in the
-                            protocol. This is preparation for a beta feature
-                            and is best not used.
+        labware definitions for labware to consider in the
+        protocol. Note that if you specify this, _only_
```
Comment (ecormany) [inline api/src/opentrons/execute.py]: RST doesn't do underlines for emphasis. 
```suggestion
        protocol. Note that if you specify this, *only*
```

File: api/src/opentrons/simulate.py [hunk 3]
```diff
@@ -242,28 +241,31 @@ def get_protocol_api(
...
+        as a ``protocols.types.APIVersion``
+        (``APIVersion(2, 0)``).
     :param bundled_labware: If specified, a mapping from labware names to
-                            labware definitions for labware to consider in the
-                            protocol. Note that if you specify this, _only_
-                            labware in this argument will be allowed in the
-                            protocol. This is preparation for a beta feature
-                            and is best not used.
+        labware definitions for labware to consider in the
+        protocol. Note that if you specify this, _only_
```
Comment (ecormany) [inline api/src/opentrons/simulate.py]: Asterisks, as above.

**Review by ecormany — APPROVED** [review]
Looks good, only improves things — although I spotted a couple more formatting changes you could add.

Maybe the rare opportunity for "style(api)" in the PR title.

File: api/src/opentrons/execute.py (hunk 2 above)
Comment (SyntaxColoring) [reply api/src/opentrons/execute.py]: Nice catch.

**Comment by SyntaxColoring** [comment]
> Maybe the rare opportunity for "style(api)" in the PR title.

oooo rare commit tag, gimme

File: api/src/opentrons/execute.py [hunk 4]
```diff
@@ -142,23 +142,23 @@ def get_protocol_api(
...
-                            labware definitions for labware to consider in the
-                            protocol. Note that if you specify this, _only_
-                            labware in this argument will be allowed in the
-                            protocol. This is preparation for a beta feature
-                            and is best not used.
+        labware definitions for labware to consider in the
+        protocol. Note that if you specify this, *only*
+        labware in this argument will be allowed in the
+        protocol. This is preparation for a beta feature
+        and is best not used.
```
Comment (jwwojak) [inline api/src/opentrons/execute.py:154]: Guys, I'm not sure where or how this text appears, but would recommend changing the last part of the sentence "and is best not used" to something like "and should not be used."

Or

Try 2 shorter sentences, "This is in preparation for a beta feature. Do not use it."

File: api/src/opentrons/simulate.py [hunk 5]
```diff
@@ -242,28 +241,31 @@ def get_protocol_api(
...
-                            labware definitions for labware to consider in the
-                            protocol. Note that if you specify this, _only_
-                            labware in this argument will be allowed in the
-                            protocol. This is preparation for a beta feature
-                            and is best not used.
+        labware definitions for labware to consider in the
+        protocol. Note that if you specify this, *only*
+        labware in this argument will be allowed in the
+        protocol. This is preparation for a beta feature
+        and is best not used.
```
Comment (jwwojak) [inline api/src/opentrons/simulate.py:253]: Another "best not used" phrase. From my suggestion above, I'd recommend changing that part of the sentence to something like "and should not be used."

Or, try 2 shorter sentences, "This is in preparation for a beta feature. Do not use it." A stronger admonition if people shouldn't use this.

**Review by jwwojak — COMMENTED** [review]
:shipit: Added some suggestions for consideration.

**Pull request merged**